src/stories/Forms/Button/index.tsx

```diff
@@ -67,6 +107,7 @@ export const Button = ({
       onClick={onClick}
       secondary={secondary}
       type={type}
+      height={size}
     >
       <ButtonLabel hide={loading}>{children}</ButtonLabel>
       {loading && (
```

Comment:
Should the ButtonStateColors be passed to the Spinner component if secondary is not `true`?

Reply:
Another good catch, only has one color so secondary is just a copy past slip from button!